### PR TITLE
Drop py27, blacken with black 20.8b0. Redo black install in CI.

### PR DESCRIPTION
**Pull request by berland**

### CODE REVIEW

File: .github/workflows/fmu-tools.yml [hunk 1]
```diff
@@ -29,28 +29,19 @@ jobs:
...
 
-    - name: Install numpy for Python 2
-      if: matrix.python-version == '2.7'
+    - name: Install requirements
       run: |
-        pip install numpy==1.16.6
+        pip install pip -U
+        pip install git+https://github.com/equinor/libecl
+        pip install git+https://github.com/equinor/fmu-ensemble
+        pip install .[tests]
```
Comment (jcrivenaes) [inline .github/workflows/fmu-tools.yml:37]: What do `pip install .[tests]` do?

Comment (berland) [reply .github/workflows/fmu-tools.yml:37]: it is the same as `pip install .` but it also picks up "extras" for `tests`as declared in `setup.py` and installs those dependencies also.

**Comment by jcrivenaes** [comment]
Most of this is also done in #53. Think I will align these PR's and just merge #53 to master if OK

File: .github/workflows/fmu-tools.yml (hunk 1 above)
Comment (jcrivenaes) [reply .github/workflows/fmu-tools.yml:37]: Nice: new to me

**Comment by jcrivenaes** [comment]
Implemented in #53

**Pull request closed without merging**